app/views/layouts/_login.html.haml

```diff
@@ -1,59 +1,2 @@
 %span.user
-  = link_to "Already been here? Log in", "#js-login-modal", class: "small subdued logout", data: {toggle: "modal"}
-.modal#js-login-modal{ tabindex: "-1", role: "dialog",
```

Comment:
I've removed this modal completely as even if we turn FB and Twitter login back on we will need to come back to this flow.